src/utils.test.tsx

```diff
@@ -292,6 +296,38 @@ describe('Utility functions', () => {
   });
 });
 
+describe('Custom Filter Functions', () => {
+  it('correctly removes seconds from date', () => {
```

Comment:
```suggestion
  it('removeSecondsFromDate correctly removes seconds from date', () => {
```
Just so its clear what it tests. The others have the function name in the describe instead.